rasa/core/channels/channel.py

```diff
@@ -244,12 +244,15 @@ def name(cls):
 
         Default implementation will just post the elements as a string."""
 
+        # we can't pass the empty "buttons" key of the message through to send_text_with_buttons()
+        kwargs.pop("buttons", None)
```

Comment:
wait what?

Reply:
it has to do with the message that gets sent in through the kwargs, as it's the whole message [here](https://github.com/RasaHQ/rasa/blob/master/rasa/core/channels/channel.py#L184) 

where the message comes from [this](https://github.com/RasaHQ/rasa/blob/3e62036bf9fd458fb5ae81d3b88079604adee357/rasa/core/actions/action.py#L111) being transfromed [here](https://github.com/RasaHQ/rasa/blob/8c18a3409d7609cb9c40be9c5107d443b80c4e24/rasa/core/events/__init__.py#L375) into a dict of all of the message data together. so the kwargs have a `"buttons: None"` arg that would get re-sent through `**kwargs` and conflict with the button argument in [here](https://github.com/RasaHQ/rasa/blob/master/rasa/core/channels/channel.py#L215)

Reply:
@akelad thoughts?

Reply:
ahh got it, sorry was occupied with other stuff :D